applications/notifications.py

```diff
@@ -55,6 +63,10 @@ class NotificationType(TextChoices):
             "application": berth_application,
             "harbor_choices": sorted(harbor_choices, key=lambda c: c.priority),
         },
+        NotificationType.BERTH_APPLICATION_REJECTED: {
+            "application": berth_application,
+            "harbor_choices": sorted(harbor_choices, key=lambda c: c.priority),
```

Comment:
We're probably not gonna need the `harbor_choices`, will depend on what information we have to include on the email.

Reply:
We don't have the the designs for the email template yet, so let's adjust this in another PR when we have it. Is that ok for you?

Reply:
Alrighty, sonunds good.